src/watson/backends.py

```diff
@@ -23,25 +23,20 @@ def regex_from_word(word):
 
 RE_SPACE = re.compile(r"[\s]+", re.UNICODE)
 
-# PostgreSQL to_tsquery operators: ! & : ( ) |
-# MySQL boolean full-text search operators: > < ( ) " ~ * + -
-RE_NON_WORD = re.compile(r'[&:"(|)!><~*+-]', re.UNICODE)
 
-
-def escape_query(text):
+def escape_query(text, chars_to_escape):
     """
     normalizes the query text to a format that can be consumed
     by the backend database
     """
     text = force_text(text)
     text = RE_SPACE.sub(" ", text)  # Standardize spacing.
-    text = RE_NON_WORD.sub(" ", text)  # Replace harmful characters with space.
+    text = chars_to_escape.sub(" ", text)  # Replace harmful characters with space.
```

Comment:
It's not really `chars_to_escape`. It's more like `re_escape_chars`.

Nitpicky, but I'd expect `chars_to_escape` to be a string.